nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -2074,7 +2074,25 @@ The Provenance Repository contains the information related to Data Provenance. T
 
 |====
 |*Property*|*Description*
-|nifi.provenance.repository.implementation|The Provenance Repository implementation. The default value is org.apache.nifi.provenance.PersistentProvenanceRepository and should only be changed with caution. To store provenance events in memory instead of on disk (at the risk of data loss in the event of power/machine failure), set this property to org.apache.nifi.provenance.VolatileProvenanceRepository.
+|nifi.provenance.repository.implementation|The Provenance Repository implementation. The default value is org.apache.nifi.provenance.PersistentProvenanceRepository.
+Two additional repositories are available as and should only be changed with caution.
```

Comment:
I am not sure '_should only be changed with caution_' is necessary. It sounds like the other two are broken or may break something. In reality they don't. They just behave different.